cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/reader/ContextConfigurationReader.java

```diff
@@ -126,7 +126,7 @@ public Map<String, Trigger> loadTriggers() {
 
     Map<String, Trigger> triggers = Maps.newHashMap();
     for (Path contextFile : this.contextConfigurations.keySet()) {
-      ContextConfiguration contextConfiguration = this.contextConfigurations.get(contextFile);
+      TemplateSetConfiguration contextConfiguration = this.contextConfigurations.get(contextFile);
```

Comment:
I'm guessing these mentions of contextConfiguration are kept for backwards compatibility?
If not you should change it to templateSetConfiguration as well for standardization